sys-apps/gentoo-systemd-integration/gentoo-systemd-integration-9999.ebuild

```diff
@@ -1,30 +1,28 @@
-# Copyright 2013-2023 Gentoo Authors
+# Copyright 2013-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit systemd udev
 
+DESCRIPTION="systemd integration files for Gentoo"
+HOMEPAGE="https://wiki.gentoo.org/wiki/Project:Systemd"
+
 if [[ ${PV} == 9999 ]]; then
 	EGIT_REPO_URI="https://anongit.gentoo.org/git/proj/gentoo-systemd-integration.git"
 	inherit autotools git-r3
 else
 	SRC_URI="https://dev.gentoo.org/~floppym/dist/${P}.tar.gz"
-	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+	KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~loong ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86"
 fi
 
-DESCRIPTION="systemd integration files for Gentoo"
-HOMEPAGE="https://wiki.gentoo.org/wiki/Project:Systemd"
-
 LICENSE="BSD"
 SLOT="0"
-IUSE=""
 
 RDEPEND="acct-group/floppy
 	acct-group/usb
 	>=sys-apps/systemd-207
-	!sys-fs/eudev
-	!sys-fs/udev"
+v"
```

Comment:
Are you sure you've run `pkgcheck scan --commits --net` (which you even have checked on the PR)?